public/scripts/main.js

```diff
@@ -109,6 +109,11 @@ function handle(e) {
     }
     $("#"+keyString.toLowerCase()).toggleClass("pressed");
   }
+  // Highlight Fn key if any of F1-F12 is pressed
+  if (e.which>=112 && e.which<=123) {
+    $("#"+e.key.toLowerCase()).addClass("pressed");
+    $("#fnc").addClass("pressed");
```

Comment:
For all of the CSS manipulations, we have an open PR, #135, which is replacing the jQuery with the native DOM functions. If you don't mind, it would be better to use those for these new lines, otherwise it will be inconsistent when both are merged later.

The native, non-jQuery code looks like this:

```js
document.querySelector("#fnc").classList.add("pressed")
```

If it's too much trouble, though, you can leave it like this and it can be fixed later.

Reply:
Done! Can you please review one more time?